src/engine/Actor.ts

```diff
@@ -878,6 +878,49 @@ export class Actor extends Entity implements Eventable, PointerEvents, CanInitia
     // Override me
   }
 
+  /**
+   * Fires every frame before collision resolution for a confirmed contact
+   * @param self
+   * @param other
+   * @param side
+   * @param contact
+   */
+  public onPreCollisionResolve(self: Collider, other: Collider, side: Side, contact: CollisionContact) {
+    // Override me
+  }
+
+  /**
+   * Fires every after collision resolution for a confirmed contact.
```

Comment:
Missing word, probably `frame`, after `Fires every`

Reply:
Oh yup, should read "fires for every collision resolution for a confirmed contact"